pkgs/applications/science/electronics/kicad/default.nix

```diff
@@ -216,6 +216,8 @@ stdenv.mkDerivation rec {
     in
     (concatStringsSep "\n"
       (flatten [
+        "runHook preInstall"
```

Comment:
what's up with this?

Reply:
Hooks need to be explicitly called by the `installPhase` script. Even if there isn't a `preInstall` hook, users may define their own using `overrideAttrs`.
https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md